### DISCUSSION TITLE
میتونم با vps ویندوزی هم اینکارو کنم؟

### DISCUSSION THREAD

**Discussion by hamidr3zo** [iranxray/hope#29] [General]
سلام. سوال من اینه که با شبیه سازی لینوکس بر روی وی پی اس ویندوزی هم میشه این برنامه رو نصب کرد؟

**Comment by iranxray** [comment]
میشه اما قطعا سر راست نخواهد بود. راحت ترین راه استفاده از همون لینوکس است.
